src/components/regions/MainSelectors.svelte

```diff
@@ -37,53 +40,43 @@ const COMPACT = true;
-            <MenuListItem  key={key} value={key}><span class='story-label
-              first'></span>{label}</MenuListItem>
-            {/each}
-        </MenuList>
-    </div>
-  </MenuButton>
-  <MenuButton tooltip={'Select an OS'} compact={COMPACT} offset={OFFSET} location='bottom' alignment='right'>
-    <div class=main-filter__label slot="label">{getFieldValueLabel('os', $store.productDimensions.os)}<div class=pull-right-edge><DownCarat size=14 /></div></div>
-    <div slot="menu">
-      <MenuList on:selection={(event) => { store.setDimension('os', event.detail.key); }}>
-        {#each CONFIG.dimensions.os.values as {key, label}, i (key)}
-          <MenuListItem key={key} value={key}><span class='story-label
-            first'></span>{label}</MenuListItem>
-          {/each}
-      </MenuList>
-    </div>
-  </MenuButton>
-  <MenuButton tooltip={'Select an Aggregation Level'} compact={COMPACT} offset={OFFSET} location='bottom' alignment='right'>
-    <div class=main-filter__label slot="label">{getFieldValueLabel('aggregationLevel', $store.productDimensions.aggregationLevel)}<div class=pull-right-edge><DownCarat size=14 /></div></div>
-    <div slot="menu">
-        <MenuList on:selection={(event) => { store.setDimension('aggregationLevel', event.detail.key); }}>
-          {#each CONFIG.dimensions.aggregationLevel.values as {key, label}, i (key)}
-          <MenuListItem key={key} value={key}><span class='story-label
-            first'></span>{label}</MenuListItem>
-          {/each}
-        </MenuList>
-    </div>
-  </MenuButton>
-  {#if $store.route.view} <!-- Hide process selector on home page. -->
-    <MenuButton tooltip={'Select a Process'} compact={COMPACT} offset={OFFSET} location='bottom' alignment='left'>
-      <div class=main-filter__label slot="label">{getFieldValueLabel('process', $store.productDimensions.process) || 'select a process'}<div class=pull-right-edge><DownCarat size=14 /></div></div>
-      <div slot="menu">
-          <MenuList on:selection={(event) => { store.setDimension('process', event.detail.key); }}>
-            {#each CONFIG.dimensions.process.values as {key, label}, i (key)}
-            <MenuListItem  key={key} value={key}><span class='story-label
-              first'></span>{label}</MenuListItem>
-            {/each}
-          </MenuList>
-      </div>
-    </MenuButton>
-  {/if}
-</div>
+{#if $store.route.section === 'probe' && $probe}
+  <div transition:fly={{ x: 5, duration: 200 }} class='main-filters'>
+    {#each Object.values(productConfig[$store.product].dimensions) as dimension, i (dimension.key)}
+      {#if dimension.values.some(
+            (di) => dimension.isValidKey === undefined
+                    || dimension.isValidKey(di.key, $probe, store),
+      )}
+      <MenuButton tooltip='Select a {dimension.title}' compact={COMPACT} offset={OFFSET} location='bottom' alignment='right'>
+        <div class=main-filter__label slot="label">
```

Comment:
`"main-filter-label"` might be better if we plan to stick with things like `"main-filter-label--active"` etc going forward otherwise we'd have to memorize when to use `_` vs `-`

Reply:
I think it was already like this so it shouldn't hold up this PR. I can do a cleanup PR for things like this later.

Reply:
Since we're roughly following hte BEM methodology, I think it should be __ for a conceptual child and -- for a variant.